shareMethod.sh

```diff
@@ -9,6 +9,7 @@ execWithTimeout() {
         echo "execWithTimeout() Command not specified"
         exit 2
     fi
+    local on_build_config_dir=$(cd $(dirname "${BASH_SOURCE[0]}");pwd)
```

Comment:
This will make ```on_build_config_dir```  equal the dir of the sh which include shareMethod.sh

but not the real on_build_config_dir

for example

a shell in on-build-config/asd/qwe/test.sh include shareMethod.sh

and then the on_build_config_dir=on-build-config/asd/qwe